frontend/src/components/header/private-elements/private-elements.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+
+import ArrowDown from '@assets/images/header/arrow-drop-down.svg';
+import DefaultAvatar from '@assets/images/header/default-avatar.png';
+
+import { IRemindersProps, Reminders } from '../reminders/reminders';
+import styles from './private-elements.module.scss';
+
+interface IPrivateComponentProps extends IRemindersProps {
+  avatar: string | undefined | null;
+}
+
+export const PrivateElements: React.FC<IPrivateComponentProps> = ({
+  favorites,
+  notifications,
+  comparisons,
+  avatar,
+}) => {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+
+  return (
+    <div className={styles.information}>
+      <Reminders
+        favorites={favorites}
+        comparisons={comparisons}
+        notifications={notifications}
+      />
+      <div
+        className={styles.avatarCover}
+        onClick={(): void => setIsMenuOpen(!isMenuOpen)}
+      >
+        <img
+          className={styles.avatar}
+          src={avatar ? avatar : DefaultAvatar}
```

Comment:
Statements that look like `smth ? smth : smth_else` must be replaced with the `smth || smth_else`